README.md

```diff
@@ -43,6 +43,7 @@ Download a zip of the code via GitHub or PyPi. Then follow the same instructions
 * AAD Username/password - Provide your AAD username and password to Kusto client (**check the notice below**).
 * AAD application - Provide app ID and app secret to Kusto client.
 * AAD code - Provide only your AAD username, and authenticate yourself using a code, generated by ADAL.
+* AZ CLI - For those already using [azure-cli](https://github.com/Azure/azure-cli), provide access token by running `az account get-access-token`.
```

Comment:
```suggestion
* AZ CLI - For those already using [azure-cli](https://github.com/Azure/azure-cli), provide access token for the logged in user`.
```

Reply:
The python impl. does not actually invoke az commands.
It inspects the az cache for the refresh token and uses it to generate tokens for other resources

Reply:
Good catch. Did not mean to imply the python implementation invokes az commands.